test/integration/kubernetes_deploy_test.rb

```diff
@@ -48,7 +48,7 @@ def test_service_account_predeployed_before_unmanaged_pod
     hello_cloud.assert_unmanaged_pod_statuses("Succeeded")
     assert_logs_match_all([
       %r{Successfully deployed in \d.\ds: ServiceAccount/build-robot},
-      %r{Successfully deployed in \d.\ds: Pod/unmanaged-pod-.*}
+      %r{Successfully deployed in \d.\ds: .*Pod/unmanaged-pod-.*}
```

Comment:
what new text is appearing here?

Reply:
This was a change before I took the PR on. Seems like it was for a fix in 1.9 but since 1.9 is back on minikube, I'll test to see if reverting this works